### ISSUE TITLE
Error passing glm:ivec as a parameter after updating to of_v0.12.0_vs_release

### ISSUE THREAD

**Issue by rjx-ray** [openframeworks/openFrameworks#7676]
We've been having issues when passing glm:ivec as a parameter after updating to of_v0.12.0_vs_release

I made this very simple app which demonstrates the issue 
[glmivecbug.zip](https://github.com/openframeworks/openFrameworks/files/12713737/glmivecbug.zip)

It has a module with just this code

	#include "glm/glm.hpp"
	#include <iostream>

	void testglm(glm::ivec2 v) {
		std::cout << v.x << "," << v.y << std::endl;
	}

and its called from ofApp::setup like this

	glm::ivec2 v = { 100, 100 };
	testglm(v);

Instead of seeing 100,100 on the console I see 1372696,0

If I include ofMain.h instead of glm/glm.hpp in glmtest.cpp it works OK and I see 100,100 on the console but that would be very undesirable for us.

So, I guess the question is, "what includes do I need to be able to use glm vectors in a module without openframeworks includes?"

**Comment by ofTheo** [comment]
If you add
‘#define GLM_FORCE_CTOR_INIT’ 

Before including glm does it then work as intended?

I think that’s the only thing we’re defining relating to glm and it has a big effect on empty contstructors ( though I don’t know why it would make a difference in your example ).

**Comment by rjx-ray** [comment]
Yes, that fixes it. We'll add it to our list of preprocessor defines. Thanks

**Comment by ofTheo** [comment]
also related #6530 
I think we'll try and add this as a preprocessor define for all the project files
